apps/tasks/lib/components/navigation_drawer.dart

```diff
@@ -73,13 +73,15 @@ class TasksNavigationDrawer extends StatelessWidget {
               title: Text(context.localization!.myTasks),
             ),
             ListTile(
+              leading: const Icon(Icons.grid_view),
               tileColor: tileColorForPage(NavigationOptions.roomoverview),
               onTap: currentPage == NavigationOptions.roomoverview
                   ? null
                   : () => pushReplace(context, RoomOverviewScreen()),
               title: Text(context.localization!.roomoverview),
             ),
             ListTile(
+              leading: const Icon(Icons.settings),
```

Comment:
either provide an Icon for all navigation options or none, maybe a clipoard with task for the MyTasks one